libpod/container_api.go

```diff
@@ -601,17 +601,69 @@ func (c *Container) Commit(pause bool, reference string, writer io.Writer, signi
 		}()
 	}
 
-	tempFile, err := ioutil.TempFile(c.runtime.config.TmpDir, "podman-commit")
+	sc := image.GetSystemContext(SignaturePolicyPath, "", false)
+	builderOptions := buildah.ImportOptions{
+		Container:           c.ID(),
+		SignaturePolicyPath: SignaturePolicyPath,
+	}
+	commitOptions := buildah.CommitOptions{
+		SignaturePolicyPath: SignaturePolicyPath,
+		ReportWriter:        writer,
+		SystemContext:       sc,
+	}
+	importBuilder, err := buildah.ImportBuilder(c.runtime.store, builderOptions)
```

Comment:
I'd like to figure out how much code ImportBuilder actually is - how much code from buildah do we really need to bring in for this? Is this a case where it's just a single file, like when we vendored in Kube stuff?

Reply:
I'll certainly look.  I was trying to avoid duplication at all costs, but if it is really small, agree that might make more sense.